model/export/AllBookletsExport.php

```diff
@@ -750,6 +746,35 @@ protected function getRows()
         return $report;
     }
 
+    private function getTotalScore(ReadableResultStorage $storage, DeliveryExecution $deliveryExecution): int
```

Comment:
The method is not an accessor, consider renaming into `calculateTotalScore()` or `extractTotalScore()`.